dist/index.js

```diff
@@ -30435,6 +30435,9 @@ const run = (inputs) => __awaiter(void 0, void 0, void 0, function* () {
     }
     if (inputs.fix) {
         const files = new Set(diagnostics.map((d) => path.join(inputs.workingDirectory, d.location.path)));
+        if (files.size == 0) {
+            return;
+        }
         const out = yield exec.getExecOutput('git', [
             'diff', '--name-only',
         ].concat([...files]), {
```

Comment:
If `files` is empty, `git diff --name-only` lists all changed files in the repository, which unrelated files can be commited.